pxtcompiler/emitter/decompiler.ts

```diff
@@ -293,12 +293,12 @@ namespace ts.pxtc.decompiler {
 
             function checkChildren(n: Node): void {
                 ts.forEachChild(n, (child) => {
-                    if (child.kind === SK.VariableDeclaration && (child as ts.VariableDeclaration).name.kind === SK.Identifier) {
-                        const name = (child as ts.VariableDeclaration).name.getText();
+                    if (ts.isDeclarationName(child)) {
```

Comment:
Note that this change expands the rename map to consider all declarations not just variable declarations. This is used in both the PY and Blocks decompiler. @riknoll , can you think of any reason why we wouldn't want all declarations?

Reply:
Nvm, I'm rolling this back so blockly decompiler only looks for variable declarations and py decompiler looks at all declarations. There were specific unit tests for blockly decomp looking for only variable renames (and not functions) so I'll leave this one alone.